IOPool/Common/test/FastMergeTestRL_cfg.py

```diff
@@ -7,6 +7,10 @@
 process = cms.Process("TESTMERGE")
 process.load("FWCore.Framework.test.cmsExceptionsFatal_cff")
 
+process.CPU = cms.Service("CPU",
+    disableJobReportOutput = cms.untracked.bool(True)
```

Comment:
There are about a dozen tests that have a reference file and compare the reference file to a job report file from the test job. I'd have to update the reference file for all the tests if I didn't disable this. And that output would do nothing to make the tests better, it would just be extraneous characters in the output and if it ever changed then I'd have to update the reference files again...

Reply:
I thought about this more last night and the reason for this is stronger than what I said yesterday. I had forgotten this. The additional text in the job report is below. It includes CPU names, speeds and counts. These will vary from machine to machine. The unit tests will be run on many different machines. This part of the job report will vary each time a different machine is used. So we can't include it in reference file comparisons. It must be excluded. The alternative is to include it in the job report but somehow remove it before the comparison. It seemed easier to just never write it in the first place than to remove it before comparing.

```
> <PerformanceReport>
>   <PerformanceSummary Metric="SystemCPU">
>     <Metric Name="CPUModels" Value="Intel Core Processor (Broadwell, IBRS)"/>
>     <Metric Name="averageCoreSpeed" Value="2194.92"/>
>     <Metric Name="cpusetCount" Value="16"/>
>     <Metric Name="totalCPUs" Value="16"/>
>   </PerformanceSummary>
> </PerformanceReport>
```

Reply:
Thanks for the clarifications